extensions/scheduler/deployment/src/test/java/io/quarkus/scheduler/test/nonblocking/NonBlockingScheduledMethodTest.java

```diff
@@ -0,0 +1,131 @@
+package io.quarkus.scheduler.test.nonblocking;
+
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.junit.jupiter.api.Assertions.fail;
+
+import java.util.List;
+import java.util.concurrent.CompletableFuture;
+import java.util.concurrent.CompletionStage;
+import java.util.concurrent.CopyOnWriteArrayList;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+import javax.enterprise.event.Observes;
+import javax.inject.Singleton;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import io.quarkus.scheduler.Scheduled;
+import io.quarkus.scheduler.ScheduledExecution;
+import io.quarkus.scheduler.SuccessfulExecution;
+import io.quarkus.test.QuarkusUnitTest;
+import io.smallrye.common.annotation.NonBlocking;
+import io.smallrye.mutiny.Uni;
+import io.vertx.core.Context;
+
+public class NonBlockingScheduledMethodTest {
```

Comment:
This might be a naive question, but I am wondering if the same test bound to work for the quartz extension? or we'll need to add the support there?

Reply:
Test added.

Reply:
Thanks!